Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+P3A.swift

```diff
@@ -199,6 +199,35 @@ extension BrowserViewController {
     }
     UmaHistogramEnumeration("Brave.Rewards.AdTypesEnabled", sample: answer)
   }
+  
+  func recordNavigationActionP3A(isNavigationActionForward: Bool) {
+    var navigationActionStorage = P3ATimedStorage<Int>.navigationActionPerformedStorage
+    var forwardNavigationActionStorage = P3ATimedStorage<Int>.forwardNavigationActionPerformed
+    
+    navigationActionStorage.add(value: 1, to: Date())
+    let newNavigationActionStorage = navigationActionStorage.maximumDaysCombinedValue
+    
+    if isNavigationActionForward {
+      forwardNavigationActionStorage.add(value: 1, to: Date())
+    }
+    
+    if newNavigationActionStorage > 0 {
+      let navigationForwardPercent = Int((Double(forwardNavigationActionStorage.maximumDaysCombinedValue) / Double(newNavigationActionStorage)) * 100.0)
+      UmaHistogramRecordValueToBucket(
+        "Brave.Toolbar.ForwardNavigationAction",
+        buckets: [
+          0,
+          .r(0..<1),
```

Comment:
Your `navigationForwardPercent` is clipped because you cast it to an `Int`, so this range will never hit

Reply:
I believe they logical solution will be removing  `.r(0..<1)` cc @iccub

Reply:
Fix at https://github.com/brave/brave-ios/pull/8054/commits/ea38e299925bd25062b56c7706aab61e5b50d090

Reply:
yes. zero alone should not be needed